ckan/pastertemplates/template/setup.py_tmpl

```diff
@@ -54,6 +54,7 @@ setup(
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    namespace_packages=['ckanext', 'ckanext.{{ project_shortname }}'],
```

Comment:
As I see it declaring ckanext.harvest, ckanext.geoview, and ckanext.dcat as namespace package is wrong and should be fixed. They aren't designed to be extended as namespace packages, and not being able to find a single concrete path for those packages (namespace packages exist in many locations) is a problem for static assets that are resolved relative to the extension path.